planex/cmd/pin.py

```diff
@@ -51,13 +51,11 @@ def repo_or_path(arg):
     (URL, None)
     """
     if arg.startswith("ssh://"):
-        split = arg.split("#")
-        if len(split) > 2 or not split:
-            raise ValueError(
-                "Expected URL or ssh://URL#commitish but got {}".format(arg))
-        if len(split) == 1:
-            return (arg, None)
-        return tuple(split)
+        if '#' in arg:
+            return tuple(arg.split("#", 1))
+
```

Comment:
If it i not there is not an error, only if there is more than `1`.

Reply:
Ah, ok. I've re-done this.